.github/workflows/version-forget-me-not.yaml

```
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: simplybusiness/version-forget-me-not@v2.3.0
+      - uses: simplybusiness/version-forget-me-not@v2.4.4
```

Please use v2 instead to save this changing regularly

```suggestion
      - uses: simplybusiness/version-forget-me-not@v2
```